qdrant-landing/content/articles/new-recommendation-api.md

```diff
@@ -0,0 +1,336 @@
+description: Qdrant 1.6 brings recommendations strategies and more flexibility to the Recommendation API.
+preview_dir: /articles_data/new-recommendation-api/preview
+social_preview_image: /articles_data/new-recommendation-api/preview/social_preview.jpg
+small_preview_image: /articles_data/new-recommendation-api/icon.svg
+weight: -80
+author: Kacper Łukawski
+author_link: https://medium.com/@lukawskikacper
+date: 2023-10-19T10:12:00.000Z
+---
+
+The most popular use case for vector search engines, such as Qdrant, is Semantic search with a single query vector. Given the 
+query, we can vectorize (embed) it and find the closest points in the index. But [Vector Similarity beyond Search](/articles/vector-similarity-beyond-search/) 
+does exist, and recommendation systems are a great example. Recommendations might be seen as a multi-aim search, where we want 
+to find items close to positive and far from negative examples. This use of vector databases has many applications, including 
+recommendation systems for e-commerce, content, or even dating apps.
+
+Qdrant has provided the [Recommendation API](https://qdrant.tech/documentation/concepts/search/#recommendation-api) 
+for a while, but the previous version had some limitations. The latest release, [Qdrant 1.6](https://github.com/qdrant/qdrant/releases/tag/v1.6.0), 
+gives you some new exciting features, including an extension of the recommendation API, making it more flexible and powerful. 
+This article unveils the internals and shows how they may be used in practice.
+
+### Recap of the old recommendations API
+
+The old [Recommendation API](https://qdrant.tech/documentation/concepts/search/#recommendation-api) in Qdrant came with some limitations. First of all, it was required to pass vector IDs for 
+both positive and negative example points. If you wanted to use vector embeddings directly, you had to either create a new point 
+in a collection or mimic the behaviour of the Recommendation API by using the [Search API](https://qdrant.tech/documentation/concepts/search/#search-api).
+Moreover, in the previous releases of Qdrant, you were always asked to provide at least one positive example. This requirement 
+was based on the algorithm used to combine multiple samples into a single query vector. It was a simple, yet effective approach. 
+However, if the only information you had was that your user dislikes some items, you couldn't use it directly.
+
+Qdrant 1.6 brings a more flexible API. You can now provide both IDs and vectors of positive and negative examples. You can even 
+combine them within a single request. That makes the new implementation backward compatible, so you can easily upgrade an existing
+Qdrant instance without any changes in your code. And the default behaviour of the API is still the same as before. However, we 
+extended the API, so **you can now choose the strategy of how to find the recommended points**.
```

Comment:
```suggestion
Qdrant 1.6 brings a more flexible API. You can now provide both IDs and vectors of positive and negative examples. You can even combine them within a single request. That makes the new implementation backward compatible, so you can easily upgrade an existing Qdrant instance without any changes in your code. And the default behaviour of the API is still the same as before. However, we extended the API, so **you can now choose the strategy of how to find the recommended points**.
```

Fix uneven sentence breaking? This is one place but there might be more which isn't visible here